src/client/rest/APIRequester.ts

```
@@ -1,4 +1,4 @@
-import Axios, { AxiosInstance, AxiosHeaders, CreateAxiosDefaults } from 'axios'
+import ky, { Options as KyOptions, SearchParamsOption } from 'ky'
```

ESLint

[error] 'SearchParamsOption' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)